conda_self/cli/main_reset.py

```diff
@@ -1,24 +1,56 @@
 from __future__ import annotations
 
+from textwrap import dedent
 from typing import TYPE_CHECKING
 
 if TYPE_CHECKING:
     import argparse
 
 HELP = "Reset 'base' environment to essential packages only."
 
+WHAT_TO_EXPECT = dedent("""
+This will:
+1.Reset your `base` to ONLY contain `conda` (and its dependencies), `conda` plugins
+and `conda-self`.
+""").lstrip()
+SUCCESS = dedent("""
+SUCCESS!
+The following operations were completed:
+
+1. Resetting of the `base` to only the essential packages and plugins.
+""")
```

Comment:
```suggestion
WHAT_TO_EXPECT = dedent(
	"""
	This will reset your `base` to ONLY contain `conda`, its plugins,
	and their dependencies.
	"""
).lstrip()
SUCCESS = dedent(
	"""
	SUCCESS!
	Reset the `base` environment to only the essential packages and plugins.
	"""
).lstrip()
```

Watch for consistency in language and formatting (not sure why ruff didn't catch this).